packages/react/src/components/Authenticator/FederatedIdentity/createFederatedIdentity.tsx

```diff
@@ -0,0 +1,16 @@
+import {
+  useHandleSignInWithRedirect,
+  FederatedIdentity,
+  FederatedIdentityInput,
+} from './types';
+import { FederatedIdentityElements } from './context/elements';
+
+export function createFederatedIdentity<
+  T extends Partial<FederatedIdentityElements>,
+  K extends string = string,
+>(
+  input: FederatedIdentityInput<T, K>
+): {
+  FederatedIdentity: FederatedIdentity<T>;
+  useHandleSignInWithRedirect?: useHandleSignInWithRedirect<K>;
```

Comment:
What is `useHandleSignInWithRedirect<K>`?

Reply:
useHandleSignInWithRedirect is a hook that allows users to track the Action State of their signInWithRedirect requests.